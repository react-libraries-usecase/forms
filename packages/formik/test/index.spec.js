import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Basic from './../src/';

describe('Basic', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Basic />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
