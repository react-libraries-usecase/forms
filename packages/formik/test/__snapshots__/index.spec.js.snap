// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic renders correctly 1`] = `
<div>
  <h1>
    Anywhere in your app!
  </h1>
  <form
    onSubmit={[Function]}
  >
    <input
      name="email"
      onBlur={[Function]}
      onChange={[Function]}
      type="email"
      value=""
    />
    <input
      name="password"
      onBlur={[Function]}
      onChange={[Function]}
      type="password"
      value=""
    />
    <button
      disabled={false}
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;

exports[`Formik renders correctly 1`] = `
.c0 {
  background: red;
  color: #fff;
  border-radius: 4px;
  cursor: pointer;
  font-size: 1rem;
  font-weight: 300;
  padding: 9px 36px;
}

<button
  className="c0"
>
  Test
</button>
`;
